content/en/docs/concepts/storage/volume-snapshots.md

```diff
@@ -120,6 +120,7 @@ spec:
   driver: hostpath.csi.k8s.io
   source:
     volumeHandle: ee0cfb94-f8d4-11e9-b2d8-0242ac110002
+  sourceVolumeMode: Filesystem # or Block. This field is only present for release-6.0 on VolumeSnapshotContent API.
```

Comment:
> This field is only present for release-6.0 on VolumeSnapshotContent API.

If I use a later release, the text implies I can only use `Filesystem`. Am I right that it's SemVer and that 6.x and any later major version support the `Block` mode?
(we should make the text unambiguous; that also helps localization teams)

Reply:
SourceVolumeMode is the mode of the volume whose snapshot is taken. It can be either “Filesystem” or “Block”. If not specified, it indicates the source volume's mode is unknown. This field is available in Volume Snapshot client v6.0.0 and higher.

Reply:
@xing-yang what change should @RaunakShah make so that the situation is clear to readers?

Reply:
I'm suggesting to add these https://github.com/kubernetes/website/pull/32673#discussion_r846635408 as comments next to the sourceVolumeMode.  Any other suggestions?